package_scripts/build_all_courses.sh

```diff
@@ -3,7 +3,6 @@
 set -euo pipefail
```

Comment:
@gumaerc Do you think we need this script anymore?

Reply:
A while back I had been using this script to build all courses locally, and even before that it was actually being used in an EC2 box to deploy the whole site before we had `ocw-studio` and Concourse. Now that we have a full dev setup for `ocw-studio` that can build the whole site locally, I think we can safely remove this without consequence.

Reply:
Unless someone wants it here, I think I'll drop that script in a separate PR.